app/alerts.py

```diff
@@ -90,15 +90,26 @@ def get_camera_positions(dpt_code=None):
         return None
 
     # We read the csv file that locates the cameras and filter for the department of interest
-    camera_positions = pd.read_csv(Path(__file__).parent.joinpath('data', 'cameras.csv'), ';')
```

Comment:
Isn't it preferable to keep the relative path with Path(__file__) instead of using ('data/cameras.csv')? I thought @frgfm  had already dealt with these

Reply:
Correct @nrslt, not sure this is still required, but at the time, it was crashing upon deployment otherwise!
Besides, `Path(__file__).parent.joinpath('data', 'cameras.csv')` is OS-independent and natively supported by python while `'data/cameras.csv'` does not work on all OS

Reply:
I didn't change it on purpose. Through the process of merging my branch to master, this occured but I'll correct it.
Thanks @frgfm for your explanation.

Reply:
No worries! What do you mean by "merging my branch to master," exactly?
If you have merging conflict on the PR, you need to merge master into your branch, that's what you did, right? :sweat_smile: